glean-core/src/internal_metrics.rs

```diff
@@ -277,6 +280,15 @@ impl DatabaseMetrics {
                 },
                 MemoryUnit::Byte,
             ),
+
+            rkv_load_result: StringMetric::new(CommonMetricData {
+                name: "rkv_load_result".into(),
+                category: "glean.error".into(),
+                send_in_pings: vec!["metrics".into()],
+                lifetime: Lifetime::Application,
```

Comment:
The metrics.yaml doesn't set a lifetime, so it defaults to `ping`, which I think makes more sense? We don't want this to be sent in every subsequent metrics ping until a restart, do we?